src/service/auth/method/repository.rs

```diff
@@ -0,0 +1,979 @@
+use std::str::FromStr;
+
+use lib::database::LocalDbClientError;
+use lib::id_gen::{ConcurrentSnowflake, ConcurrentSnowflakeError};
+use uuid::Uuid;
+
+use gen::database::{
+    FunAuthAuthenticateReq, FunAuthAuthenticateResp, FunAuthAuthenticateRespRow,
+    FunAuthAuthorizeReq, FunAuthAuthorizeResp, FunAuthAuthorizeRespRow,
+    FunAuthBasicAuthenticateReq, FunAuthBasicAuthenticateResp, FunAuthBasicAuthenticateRespRow,
+    FunAuthChangePasswordReq, FunAuthChangePasswordResp, FunAuthChangePasswordRespRow,
+    FunAuthGetPasswordSaltReq, FunAuthGetPasswordSaltResp, FunAuthGetPasswordSaltRespRow,
+    FunAuthGetRecoveryQuestionsReq, FunAuthGetRecoveryQuestionsResp,
+    FunAuthGetRecoveryQuestionsRespRow, FunAuthResetPasswordReq, FunAuthResetPasswordResp,
+    FunAuthResetPasswordRespRow, FunAuthSetRecoveryQuestionsReq, FunAuthSetRecoveryQuestionsResp,
+    FunAuthSetRecoveryQuestionsRespRow, FunAuthSetTokenReq, FunAuthSetTokenResp,
+    FunAuthSetTokenRespRow, FunAuthSignupReq, FunAuthSignupResp, FunAuthSignupRespRow,
+    FunGetRecoveryQuestionDataReq, FunGetRecoveryQuestionDataResp,
+    FunGetRecoveryQuestionDataRespRow, FunSubmitRecoveryAnswersReq, FunSubmitRecoveryAnswersResp,
+    FunSubmitRecoveryAnswersRespRow,
+};
```

Comment:
`use gen::database::*`

Reply:
Thank you, fixed and pushed to local-storage branch